pkgs/development/tools/vcpkg-tool/default.nix

```
@@ -0,0 +1,88 @@
+{ autoconf
+, automake
+, bash
+, cacert
+, cmake
+, cmakerc
+, coreutils
+, curl
+, fetchFromGitHub
+, fmt
+, gcc
+, git
+, gnumake
+, gzip
+, jq
+, lib
+, makeWrapper
+, meson
+, ninja
+, perl
+, pkg-config
+, python3
+, runtimeShell
+, stdenv
+, vcpkg-tool
+, zip
+, zstd
+}:
+let
+  # These are the most common binaries used by vcpkg
+  # If a port requires a binary that is not in this list,
+  # it can be added by an overlay
+  runtimeDeps = [
+    autoconf
+    automake
+    bash
+    cacert
+    coreutils
+    curl
+    cmake
+    gcc
+    git
+    gnumake
+    gzip
+    meson
+    ninja
+    perl
+    pkg-config
+    python3
+    zip
+    zstd
+  ];
+in
+stdenv.mkDerivation {
+  pname = "vcpkg-tool";
+  version = "2023-06-22";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "vcpkg-tool";
+    rev = "2023-06-22";
+    hash = "sha256-/Bn2TW6JWU676NzsQdtC6G513MtRHblLVP0pK5jyCWc=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    cmakerc
+    fmt
+    ninja
```

```suggestion
    ninja
    makeWrapper
```

Is missing which causes build to fail in post-fixup